cli/internal/core/auth/add.go

```diff
@@ -36,12 +36,7 @@ func (ivkr _adder) Add(
 	username string,
 	password string,
 ) error {
-	nodeHandle, err := ivkr.nodeProvider.CreateNodeIfNotExists()
```

Comment:
This is now handled by the lazylocalnode wrapper when using an api client based OpNode.